README.md

```diff
@@ -32,11 +32,13 @@ We will now cover the installation of all of these. BTW, I'm assuming you're an
 
 ### PostgreSQL
 
-We will use `brew` to install PostgreSQL 9.5. There is one important bit though, we must specify `--with-python` to make sure we get that `plypythonu` extension we will later need.
+We will use `brew` to install PostgreSQL 9.5 and uninstall previous versions (if any). There is one important bit though, we must specify `--with-python` to make sure we get that `plypythonu` extension we will later need.
 
 ```
 brew tap Homebrew/homebrew-versions
+brew remove postgresql
 brew install homebrew/versions/postgresql95 --with-python
+brew link postgresql95
```

Comment:
You had to `link` because you had previously installed it, but it should be `install`. Maybe I'd focus this differently:

If you have a previous version of PostgreSQL, make sure to migrate it. See `brew info postgresql` for details. If you decide to uninstall it completely, run `brew remove postgresql`. When you have sorted that out:
```
brew install homebrew/versions/postgresql95 --with-python
brew install postgresql95
```